x-pack/plugins/fleet/public/components/enrollment_instructions/manual/index.tsx

```diff
@@ -80,7 +80,7 @@ sudo elastic-agent enroll ${enrollArgs} \nsudo systemctl enable elastic-agent \n
 
   const googleCloudShellCommand = `gcloud config set project ${gcpProjectId} && ${
     gcpAccountType === 'organization-account' ? `ORG_ID=${gcpOrganizationId}` : ``
-  } FLEET_URL=${fleetServerUrl?.trim()} ENROLLMENT_TOKEN=${enrollmentToken} STACK_VERSION=${agentVersion} ./deploy.sh`;
+  } FLEET_URL=${fleetServerUrl?.trim()} ENROLLMENT_TOKEN=${enrollmentToken} STACK_VERSION=${agentVersion} ./deploy_compute.sh`;
```

Comment:
We need to think how to handle this change in serverless. if I understand correctly the change in cloudbeat will be available only after the next minor release, eg. in 8.14.0. That means in the servelress Kibana will reference to `./deploy_compute.sh` while it won't be available on the cloudbeat artifact (in the downloaded Elastic Agent).